lib/scala/pkg/src/test/scala/org/enso/pkg/ConfigSpec.scala

```diff
@@ -93,6 +93,16 @@ class ConfigSpec
       val serialized = parsed.toYaml
       serialized should include("edition: '2020.1'")
     }
+
+    "correctly de-serialize missing edition" in {
```

Comment:
Maybe rename it like this?

```suggestion
    "correctly de-serialize empty edition field" in {
```

IMO "missing" suggests that the field was not specified at all, not that it is specified to be empty. And that 'missing' scenario is tested by "only require the name and use defaults for everything else".